barberia/src/App/pages/covertdate.js

```diff
@@ -0,0 +1,29 @@
+export function convertDate(str) {
```

Comment:
or you can get rid of this function by using moment to convert the date, your choice either keep this or use moment